### ISSUE TITLE
syntax error near "(JSON_OBJECT())

### ISSUE THREAD

**Issue by Panlq** [kyleconroy/sqlc#2317]
### Version

Other

### What happened?

version 1.11.0
The JSON field cannot be set with a default value！

run sqlc generate failed

```shell
# package generated
schema/test.sql:xx:34: syntax error near "(JSON_OBJECT()) COMMENT 'status',"
````

### Database schema

```sql
CREATE TABLE `history` (
  `id` char(36) CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci NOT NULL DEFAULT '' COMMENT 'id',
  `job_status` json DEFAULT (JSON_OBJECT()) COMMENT 'status',
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci COMMENT='execute_history';
```

**Comment by kyleconroy** [comment]
I'm not sure when it was fixed, but this works in v1.18.0 

https://play.sqlc.dev/p/b6338b2996ac3eb526bca9309f85d16d8f4179f6a04b1ddaf52c4acb1e58cbca

Please upgrade to the latest version.

**Comment by Panlq** [comment]
> I'm not sure when it was fixed, but this works in v1.18.0
> 
> https://play.sqlc.dev/p/b6338b2996ac3eb526bca9309f85d16d8f4179f6a04b1ddaf52c4acb1e58cbca
> 
> Please upgrade to the latest version.

First of all, thank you very much, sqlc is very useful, are there any breaking cross-version upgrades that we have used in production, will there be a big difference in the upgrade version
